### PR TITLE
AEP interpolation and blending

### PR DESCRIPTION
**Pull request by kieranricardo**

### CODE REVIEW

File: scripts/ari_interpolate.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    grid_params[param_names[j]] = z
+
+
+# combine the AEP for TC, synoptic and TS
+shape, scale = grid_params["shape"][:, :, None], grid_params["scale"][:, :, None]
+rate, mu = grid_params["rate"][:, :, None], grid_params["mu"][:, :, None]
+
+print("max windspeed:", hazard.GPD.gpdReturnLevel(10_000, mu, shape, scale, rate).max())
+
+tc_aep = 1.0 / gdp_recurrence_intervals(windspeeds[None, None, :], mu, shape, scale, rate).data
```
Comment (wcarthur) [inline scripts/ari_interpolate.py]: aep = 1.0 - np.exp(-1.0/gpd_recurrence_intervals)

File: scripts/ari_interpolate.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+# convert windspeed coords + AEP values to AEP coords and windspeed values
+# funky code to quickly linearly interpolate (a for loop + numpy interpolate was much much slower)
+ris = np.array([
+    2, 3, 4, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 100,
+    150, 200, 250, 300, 350, 400, 450, 500, 1000, 2000, 2500,
+    5000, 10000]
+)
+
+aeps = 1 / ris
```
Comment (wcarthur) [inline scripts/ari_interpolate.py]: aep = 1.0 - np.exp(-1.0/ris)

**Review by wcarthur — CHANGES_REQUESTED** [review]
Check definition of the AEP in ari_interpolate.py. Will be a subtle difference, but best to be correct from outset

**Comment by kieranricardo** [comment]
@wcarthur nice catch, I shouldn’t have assumed that AEP = 1 / ARI! this approach makes a lot more sense

**Review by wcarthur — APPROVED** [review]

**Pull request merged**